politeiawww/backend.go

```diff
@@ -596,14 +609,14 @@ func (b *backend) ProcessLogin(l www.Login) (*www.LoginReply, error) {
 	err = bcrypt.CompareHashAndPassword(user.HashedPassword,
 		[]byte(l.Password))
 	if err != nil {
-		return nil, userError{
+		return nil, 0, userError{
 			errorCode: www.StatusInvalidEmailOrPassword,
 		}
 	}
 
 	reply.IsAdmin = user.Admin
 	reply.ErrorCode = www.StatusSuccess
-	return &reply, nil
+	return &reply, user.ID, nil
```

Comment:
Shouldn't the User ID just be passed in via `LoginReply` so that the UI can use it to differentiate the current user's proposals and comments from everyone else's? It should also be passed into the `MeReply`. Maybe we should even have a common `UserDetails` structure that can be reused for both of these commands.

Reply:
Yeah I struggled with that but I took your recommendation.